### PR TITLE
feat: [CDS-39172]: help panel for pipeline studio

### PR DESCRIPTION
**Pull request by dhruvUpadhyay07**
### Summary
integerated help panels for pipelien studio screens , input set , triggers and execution history page

#### Screenshots

https://user-images.githubusercontent.com/64455881/177117824-bc4110a3-4ae5-4cba-9c68-96a0ee2d4089.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>
- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>
- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by dhruvUpadhyay07** [comment]
retrigger titlecheck

File: src/index.html [hunk 1]
```diff
@@ -69,9 +69,9 @@
     <% if (__DEV__) { %>
     <script>
       window.deploymentType = 'SAAS'
-      window.helpPanelAccessToken = '5EHT-ZBWYjweBLS62PNKf-vp3ZbXAylRUlPRPcHwqAg'
+      window.helpPanelAccessToken = 'iMDhj0H7-zbCwnAe4avVNNFBni6-dZr4vLcaRNi5UUY'
```
Comment (vkbansal) [inline src/index.html:72]: This token should come from ENV variabales

Comment (dhruvUpadhyay07) [reply src/index.html]: pushed this file by mistake, reverted

**Comment by dhruvUpadhyay07** [comment]
retrigger coverage

**Comment by dhruvUpadhyay07** [comment]
retrigger sonar

**Review by kapilkumawat86 — APPROVED** [review]

**Review by vkbansal — DISMISSED** [review]

**Comment by dhruvUpadhyay07** [comment]
trigger cypress

**Comment by dhruvUpadhyay07** [comment]
trigger cypress

**Comment by dhruvUpadhyay07** [comment]
trigger cypress

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**